client/gui-qt/helpdlg.cpp

```diff
@@ -1359,7 +1359,7 @@ void help_widget::set_topic_terrain(const help_item *topic,
       fc_snprintf(buffer, sizeof(buffer),
                   PL_("%d turn", "%d turns", pterrain->cultivate_time),
                   pterrain->cultivate_time);
-      str = N_("Irrig. Rslt/Time:");
+      str = N_("Cultivate Rslt/Time:");
```

Comment:
Does checking the `irrigation_result` in the `if` still make sense?

Reply:
Based on my limited experience with the code I think so. I was mostly after changing the string value to fix the bug and get rid of "Trans." to a fully spelled out "Transform".

Reply:
Convinced by the multiple tests shown privately.